src/main/java/io/redlink/more/data/event/DeregistrationSpringEvent.java

```diff
@@ -0,0 +1,17 @@
+package io.redlink.more.data.event;
+
+import io.redlink.more.data.model.RoutingInfo;
+import org.springframework.context.ApplicationEvent;
+
+public class DeregistrationSpringEvent extends ApplicationEvent {
```

Comment:
Ich hätte ein 

```
public class ParticipantUpdateEvent extends ApplicationEvent {}
```
Mit studyId, participantId und enum Action gemacht

RoutingInfo ist glaube ich eine generierte Klasse. Die würde ich nie in einer internen API verwenden